specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/2024-08-22-preview/confidentialledger.json

```diff
@@ -0,0 +1,1668 @@
+          "$ref": "#/definitions/ForwardingRequired"
+        },
+        "interpreter_reuse": {
+          "$ref": "#/definitions/InterpreterReusePolicy"
+        },
+        "js_function": {
+          "type": "string"
+        },
+        "js_module": {
+          "type": "string"
+        },
+        "mode": {
+          "$ref": "#/definitions/Mode"
+        },
+        "openapi": {
+          "$ref": "#/definitions/json"
+        },
+        "openapi_hidden": {
+          "$ref": "#/definitions/boolean"
+        },
+        "redirection_strategy": {
+          "$ref": "#/definitions/RedirectionStrategy"
+        }
+      },
+      "required": [
+        "forwarding_required",
+        "authn_policies"
+      ],
+      "type": "object"
+    },
+    "json": {},
+    "json_array": {
+      "items": {
+        "$ref": "#/definitions/json"
+      },
+      "type": "array"
+    },
+    "ForwardingRequired": {
+      "enum": [
+        "sometimes",
+        "always",
+        "never"
+      ],
+      "x-ms-enum": {
+        "name": "ForwardingRequired",
+        "modelAsString": true
+      },
+      "type": "string"
+    },
+    "InterpreterReusePolicy": {
+      "properties": {
+        "key": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "key"
+      ],
+      "type": "object"
+    },
+    "Mode": {
+      "enum": [
+        "readwrite",
+        "readonly",
+        "historical"
+      ],
+      "x-ms-enum": {
+        "name": "Mode",
+        "modelAsString": true
+      },
+      "type": "string"
+    },
+    "RedirectionStrategy": {
+      "enum": [
+        "none",
+        "to_primary",
+        "to_backup"
+      ],
+      "x-ms-enum": {
+        "name": "RedirectionStrategy",
+        "modelAsString": true
+      },
+      "type": "string"
+    },
+    "ModuleDef": {
+      "properties": {
+        "module": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "name",
+        "module"
+      ],
+      "type": "object"
+    },
+    "ModuleDef_array": {
+      "items": {
+        "$ref": "#/definitions/ModuleDef"
+      },
+      "type": "object"
+    },
+    "JSRuntimeOptions": {
+      "description": "JS runtime options for user defined endpoints",
+      "properties": {
+        "log_exception_details": {
+          "$ref": "#/definitions/boolean"
+        },
+        "max_cached_interpreters": {
+          "$ref": "#/definitions/uint64"
+        },
+        "max_execution_time_ms": {
+          "$ref": "#/definitions/uint64"
+        },
+        "max_heap_bytes": {
+          "$ref": "#/definitions/uint64"
+        },
+        "max_stack_bytes": {
+          "$ref": "#/definitions/uint64"
+        },
+        "return_exception_details": {
+          "$ref": "#/definitions/boolean"
+        }
+      },
+      "type": "object"
+    },
+    "uint64": {
+      "maximum": 18446744073709552000,
```

Comment:
This line is causing the PR tooling to crash. Further, this value is actually larger than possible in an unsigned 64 bit value
```
2^64-1 = 18,446,744,073,709,551,615
```
and if the intent is to allow any unsigned 64-bit value then specifying a maximum is unnecessary, so best to just remove it.
```suggestion
```